paseos/paseos.py

```diff
@@ -92,9 +92,48 @@ def add_known_actor(self, actor: BaseActor):
         # Else add
         self._known_actors[actor.name] = actor
 
+    @property
+    def local_actor(self) -> BaseActor:
+        """Returns the local actor.
+
+        Returns:
+            BaseActor: Local actor
+        """
+        return self._local_actor
+
+    @property
+    def known_actors(self) -> dict:
+        """Returns known actors.
+
+        Returns:
+            dict of BaseActor: Dictionary of the known actors.
+        """
+        return self._known_actors
+
+    @property
+    def known_actor_names(self) -> list:
+        """Returns names of known actors.
+
+        Returns:
+            list: List of names of known actors.
+        """
+        return self._known_actors.keys()
+
     def emtpy_known_actors(self):
+        """Clears the list of known actors."""
         self._known_actors = {}
 
+    def remove_known_actor(self, actor_name: str):
```

Comment:
Right now we are only distinguishing actors based on their names. I think it would make sense to make this a bit more general and use the a hash of the static features in the actors. I will write an issue for this.